hydromt_wflow/wflow_sediment.py

```diff
@@ -309,17 +309,17 @@ def setup_lulcmaps(
         lulc_fn: Union[str, Path, xr.DataArray],
         lulc_mapping_fn: Union[str, Path, pd.DataFrame] = None,
         planted_forest_fn: Union[str, Path, gpd.GeoDataFrame] = None,
-        lulc_vars: List[str] = [
-            "landuse",
-            "Cov_River",
-            "Kext",
-            "N",
-            "PathFrac",
-            "Sl",
-            "Swood",
-            "USLE_C",
-            "WaterFrac",
-        ],
+        lulc_vars: Dict = {
+            "landuse": None,
+            "Cov_River": None,
+            "Kext": "input.vertical.kext",
+            "N": None,
+            "PathFrac": "input.vertical.pathfrac",
+            "Sl": None,
+            "Swood": "input.vertical.storage_wood",
+            "USLE_C": "input.vertical.usleC",
+            "WaterFrac": None,
+        },
```

Comment:
@hboisgon, is it indeed correct that these parameters (`lateral.land.n`, `vertical.storage_leaf` and `vertical.waterfrac`) are not needed/specified for wflow_sediment?
I now set their keys to None otherwise the test fails, but wanted to make sure this is indeed correct

Reply:
I checked in the wflow sediment code and for sure ``lateral.land.n``,  ``vertical.waterfrac`` are not used.8 ``vertical.storage_leaf`` should though. For ``Cov_river`` I see it is not used anymore as it was found the equation was not doing too well. Maybe I would like to add it again in the model at some point so we can keep the values in the mapping table but remove it from the list of params to prepare?

Reply:
Can you then also update the docstring similar to the wflow one?